.travis.yml

```diff
@@ -6,6 +6,7 @@ env:
     - PKG_RELEASE=$(python ${DIST_REPO}/scripts/get-version-release.py --release)
     - PKG_RELEASE_EL7=${DIST_REPO}/rpms/build/f5-openstack-agent-${PKG_VERSION}-${PKG_RELEASE}.el7.noarch.rpm
     - PKG_RELEASE_1404=${DIST_REPO}/deb_dist/python-f5-openstack-agent_${PKG_VERSION}-${PKG_RELEASE}_1404_all.deb
+    - RELEASE_TAG="v$(echo $PKG_VERSION | cut -d . -f 1,2)"
```

Comment:
Understanding that this is a 'picky' semantic change, I apologize at the level of 'meaning' at runtime; however, there's no reason to use the `"`.  Compile this with the fact that we're not using `"` anywhere else in these definitions without a clear reason, and I'm left with the thought that we should continue to use the same pattern and only use quotes when they're necessary.

Proof that they're not necessary:
```shell
$ zoo=v$(echo 8.3.2b1 | cut -d . -f 1,2)
$ echo $zoo
v8.3
```
Depending on your history, this may be cringe-worthy; however, the decision to not use quotes has already been made.

Reply:
Fixed. :)